### ISSUE TITLE
Presentation criteria for Friday

### ISSUE THREAD

**Issue by CYPIAPT-admin** [gabrielperales/formulationTool#79]
Each team will have 15minutes to present, and it should involve both the product owner/s and devs addressing the following:

Implementation – how challenging will it be to begin to use for young people/in services? Has the product owner begun to locate people/services to trial it with?
Technical viability – will it be possible to complete the technical elements of the product with the resources/time available?
Technical spec – I.e. Platform/format, data security

Need – is there a need for this, i.e. is it a real world problem for others?
Evidence-base – have you considered the evidence base for what you are doing, i.e. is there evidence that the functions of your app will improve emotional wellbeing and mental health?
Product ownership – Can the product owner continue to oversee and guide development?
UX feedback – what do the people who’ve used the working prototype think of it, etc? How will their feedback be utilised?
Product roadmap – e.g. What would your plans for developing product be if you won?

On the night we will explain to the audience that these are the parameters on which we are assessing the products, and that they should consider these questions when voting.

The audience will vote through mentimeter but will not determine who will win. The panel will decide on the winners and take the audience vote into account.

**Comment by cnludlow** [comment]
@gabrielperales @mcelearr @Shadows666 @TroyMaeder I'm currently working on a presentation to address the clinical questions.

**Comment by mcelearr** [comment]
@cnludlow Good stuff. Can we review the presentation as it stands today or tomorrow? Try and make it clear if there are any gaps (for example your question on data protection options) and we will try to fill in the blanks.

**Comment by cnludlow** [comment]
@mcelearr Yes, happy to review the presentation. I am going to be working on it today. I think I need to spend the first 5 mins of the presentation talking about the rationale and trying to get the audience to understand what we are trying to do. How do you want to review the presentation? Do you want me to visit you guys in Bethnal Green tomorrow?

**Comment by cnludlow** [comment]
@mcelearr @gabrielperales @Shadows666 @TroyMaeder. Wondering if people want to practice the presentation? I'm free to meet any time Friday or we could meet before the presentation for a practice run at Anna Freud (e.g., around 4pm). What do people think? Otherwise, I'm happy to share what I have through GitHub.

**Comment by TroyMaeder** [comment]
@cnludlow sounds like you've been getting some work done 👍  I'm happy to do whatever works best for the rest of the team : )

**Comment by gabrielperales** [comment]
@cnludlow I'm with @TroyMaeder, I'm also happy to do whatever works best for the rest of the team

**Comment by cnludlow** [comment]
@TroyMaeder @gabrielperales. @mcelearr @Shadows666 Well, maybe we could meet around 430pm tomorrow for a practise run somewhere at the Anna Freud Centre? That would give us at least a solid hour before the presentation starts at 630pm? It might be nice to have 30 mins off from 6-630pm before the event start, but it might also give us some emergency time up our sleeves to iron out any problems that might pop up.

**Comment by CYPIAPT-admin** [comment]
@TroyMaeder @gabrielperales. @mcelearr @Shadows666 @cnludlow  If you want to practice here (Rodney street), I can book you some space, and will be around if you have any last minute implementation questions (especially around use and embedding specific technologies within services

**Comment by cnludlow** [comment]
@CYPIAPT-admin Yes, that sounds great. We would really appreciate a room booking at Rodney Street. Thank you.

**Comment by mcelearr** [comment]
@cnludlow @CYPIAPT-admin that all sounds good. I can meet tomorrow at 4:30pm at Rodney Street

**Comment by TroyMaeder** [comment]
Does 4:30pm work for everyone then? @gabrielperales @Shadows666

**Comment by cnludlow** [comment]
@CYPIAPT-admin Just wondering what the 15 mins entails? Do we only have 15 mins to talk overall for the whole night? Or is the 15 mins just for answering the questions you posted (i.e., information, technical viability etc). 

I need to be able to explain the rationale of the software to the audience. Because it is a technical piece of software designed for service-improvement, it tends to go over people's heads without an adequate explanation. I think I would need 10 mins just to explain the rationale alone. Otherwise, people won't understand what the software is and what it is trying to do.

**Comment by CYPIAPT-admin** [comment]
@cnludlow I apologise, but due to constraints on how long we can stay in the building and in the essence of fairness we are having to cap each team to just 15 minutes
